docs/docs/modules/indexes/retrievers/supabase-hybrid.mdx

```diff
@@ -29,7 +29,7 @@ create table documents (
 -- Create a function to similarity search for documents
 create function match_documents (
   query_embedding vector(1536),
-  match_count int,
+  match_count int DEFAULT null,
```

Comment:
Is it possible to avoid this? 

We need it for advanced filtering because it basically filters down retrieved documents after they're initially fetched from Supabase? Is this really different then from setting a really high `k` and just using a `.filter()` method in JS?

Reply:
The problem I've had was that without that default value, the similaritySearch function doesn't return the correct number of results since the procedure already limits the results before it even gets to the filter. But yeah it is possible to just set the match_count really high and then add limit in the filter itself. I suppose if we include that in the example, we don't have to add the default value.